test/oph/ehoks/palaute/handler_test.clj

```diff
@@ -101,16 +101,18 @@
             (let [resp (post! palaute-app
                               "/tyoelamapalaute/7/vastaajatunnus")
                   data (:data (test-utils/parse-body (:body resp)))]
-              (is (= (:status resp) 200))
+              (is (= (:status resp) 200) data)
               (is (not (nil? (:vastaajatunnus data))))
+              ;; TODO: test here that the palaute is synced to DDB with
+              ;; hankkimistapa-id
```

Comment:
Jonkinlainen testi tälle löytyy nykyään: testi `test-handle-all-palautteet-waiting-for-vastaajatunnus` nimiavaruudessa `oph.ehoks.palaute.tyoelama_test`.

Reply:
pointti on siis se, että saatoin tässä kohtaa rikkoa sen että hankkimistapa-id tulee mukana ja siksi laitoin itselleni muistiin.